bin/import_cv2.py

```diff
@@ -53,23 +53,23 @@ def _preprocess_data(audio_dir, tsv_dir):
 def _maybe_convert_set(audio_dir, input_tsv):
     output_csv =  path.join(audio_dir,os.path.split(input_tsv)[-1].replace('tsv', 'csv'))
     print("Saving new DeepSpeech-formatted CSV file to: ", output_csv)
-    
+
     # Get audiofile path and transcript for each sentence in tsv
     samples = []
     with open(input_tsv) as input_tsv_file:
         reader = csv.DictReader(input_tsv_file, delimiter='\t')
         for row in reader:
             samples.append((row['path'], row['sentence']))
-            
+
     # Keep track of how many samples are good vs. problematic
     counter = { 'all': 0, 'too_short': 0, 'too_long': 0 }
     lock = RLock()
     num_samples = len(samples)
     rows = []
-    
+
     def one_sample(sample):
         """ Take a audio file, and optionally convert it to 16kHz WAV """
-        mp3_filename = path.join(audio_dir, sample[0])
+        mp3_filename = path.join(audio_dir, sample[0]) + ".mp3"
```

Comment:
That's clearly wrong.

Reply:
@lissyx how? I have this locally and it works

Reply:
@lissyx - the released CV doesn't have extensions for some reason

Reply:
@JRMeyer Beta had extensions, it makes no sense to not have extensions ...

Reply:
@qboot Can you make it at least with a check, it really really feels bogus that CV suddenly changed the naming convention.

Reply:
@lissyx Why it is wrong? Current .tsv files in the Common Voice dataset doesn't have filename extensions, just name. So when you try to convert mp3 files to wav files sting 72 `mp3_filename = path.join(audio_dir, sample[0])` doesn't include .mp3 extension, and converting doesn't work.

Reply:
@kremnik It's wrong because that's not what should be in the dataset: all the beta releases were fine with `.mp3`.